### PR TITLE
Feature/5 basic tag rules

### PR DESCRIPTION
**Pull request by jasrial**

### CODE REVIEW

**Comment by jasrial** [comment]
es sind jetzt wieder commits von vorher drinnen -.-

**Comment by jasrial** [comment]
es sind jetzt wieder die commits von vorher drinnen -.- aber ich glaube das kommt davon dass ich jetzt wieder von der develop branch ausgegangen bin

File: src/components/Rule.css [hunk 1]
```diff
@@ -1,3 +1,10 @@
+<<<<<<< HEAD
+.Rule {
+  margin: 10px;
+}
+
+=======
+>>>>>>> e6e05e736dae6b022ac304ced666ff7e36c2db0f
```
Comment (vlabs) [inline src/components/Rule.css]: this is not right

File: src/components/TagRules.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+const rowStyle = {
+  marginTop: 50
+};
+
+const TagRules = props => (
+  <div>
+    <Container fluid>
+      <Row style={rowStyle}>
+        <Col>
+          <p style={{ fontSize: 30 }}>Dimension Ruleset 1</p>
```
Comment (vlabs) [inline src/components/TagRules.js]: do not use style but a css class

File: src/components/TagRules.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import Ruleset from "./Ruleset";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+import { Container, Row, Col, Button } from "react-bootstrap";
+
+const rowStyle = {
+  marginTop: 50
```
Comment (vlabs) [inline src/components/TagRules.js]: same here

File: src/components/Rule.css (hunk 1 above)
Comment (jasrial) [reply src/components/Rule.css]: f236198fcc753cc5fc558bcdad4a9c6d933f5d94

File: src/components/TagRules.js (hunk 3 above)
Comment (jasrial) [reply src/components/TagRules.js]: 7a6a0fa08d4d2087669c1e0dc00659e8ec623f20

File: src/components/TagRules.js (hunk 2 above)
Comment (jasrial) [reply src/components/TagRules.js]: 7a6a0fa08d4d2087669c1e0dc00659e8ec623f20

Comment (vlabs) [reply src/components/TagRules.js]: this has not been addressed!

Comment (jasrial) [reply src/components/TagRules.js]: 62589e46dfdd4080c81608764884e51a65a5b55b

sry my bad, i misread and thought you meant the `<Row>` statement!

File: src/components/Ruleset.js [hunk 4]
```diff
@@ -9,21 +9,21 @@ import { faTrash } from "@fortawesome/free-solid-svg-icons";
 const Ruleset = props => (
   <div className="Ruleset">
     <Container fluid>
-      <Row className="Row">
+      <Row className="SetRow">
         <Col md={3}>
           <p style={{ fontWeight: "bold" }}>For source dimension:</p>
```
Comment (vlabs) [inline src/components/Ruleset.js]: also get rid of this `style` attribute (eg by a bootstrap heading)

explanation:
1) we wanna avoid css, bootstrap should cover it for us (see https://getbootstrap.com/docs/4.3/content/typography/)
2) if we really need css, use css classes and not inlines styles (via `style` atttribute)

please check throughout the entire project

Comment (jasrial) [reply src/components/Ruleset.js]: 83ce494ffd5fbbbb81b61b6c5c1ca8ecd13dafe6

**Pull request closed without merging**